datapackage_pipelines_budgetkey/pipelines/budget/national/changes/processed/pipeline-spec.yaml

```diff
@@ -199,6 +199,8 @@ national-budget-changes-aggregated:
             es:itemType: date
           pending:
             es:itemType: boolean
+          year:
```

Comment:
why didn't you use the `date` field above?

Reply:
Looking at the data, it seems not all rows have a date, also it's an array

I thought it's safer to use year and try to fix later in #112 to improve granularity